### ISSUE TITLE
The method 'substring' was called on null. Error coming in drawer. 

### ISSUE THREAD

**Issue by SapinderSingh** [infiniteoverflow/Libro#132]
**Describe the bug**
If user don't logs in using google, we don't have user's name, but we have to display something on the drawer, if we don't have user's name. 

**To Reproduce**
1. Login to app using email and password.
2. Click on drawer icon in the top left. 
3. See error

**Expected behavior**
Either there should be user displayed displayed, or something like, 'nothing' should be displayed, rather than an error.
### Screenshot 
![2121](https://user-images.githubusercontent.com/29424616/118391645-fcd01180-b652-11eb-9c8a-75aaa2e69ee2.jpg)

**Comment by infiniteoverflow** [comment]
I will just check and get back to u on this.

**Comment by infiniteoverflow** [comment]
I am not facing this error in my device. Can you please pull the new changes and check again.

**Comment by SapinderSingh** [comment]
@infiniteoverflow Please watch this video.
https://user-images.githubusercontent.com/29424616/118956257-120fad80-b97d-11eb-9bde-4422538301a4.mp4

Note that this error will only come if you sign in using email and password, it will not come, if you login using google.

**Comment by EAsmithaShrree** [comment]
Heyy so I coded this part of the program. I think you will receive this error if you had signed up to the app before I added the code for asking for the username during signup. Because before I added that part of the code, everybody who signed up would have done so without giving their username so that will be null for all the old users. 

Does this make sense? I am guessing that this is the problem.
